FrontEnd/myapp/tailwind.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
would like some explanation of this

Reply:
good catch, I am updating with a deleted version of this. Originally, I was going to use tailwind as our navbar service but found the default boostrap one easier. Thank you.